samples/snippets/fsharp/azure/table-storage.fsx

```diff
@@ -1,111 +1,90 @@
+#r "nuget:Azure.Data.Tables" // Load the Azure Data Tables nuget package
+
 open System
 open Azure
 open Azure.Data.Tables // Namespace for Table storage types
-open System.Linq
-open System.Collections.Generic
 
 //
 // Get your connection string.
 //
 
-let storageConnString = "..." // fill this in from your storage account
+let storageConnString = "UseDevelopmentStorage=true" // fill this in from your storage account
 
 //
 // Create the Table service client.
 //
 
 // Create the table client.
-let tableClient = TableServiceClient(storageConnString)
+let tableClient = TableServiceClient storageConnString
 
 //
 // Create a table.
 //
 
 // Retrieve a reference to the table.
-let table = tableClient.GetTableClient("people")
+let table = tableClient.GetTableClient "people"
 
 // Create the table if it doesn't exist.
-ignore(table.CreateIfNotExists())
+table.CreateIfNotExists() |> ignore
 
 //
 // Add an entity to a table. The last name is used as a partition key.
 //
 
 type Customer(firstName, lastName, email: string, phone: string) =
     interface ITableEntity with
```

Comment:
Would it be worth considering a comment here to indicate that the shadowing is needed because serialization doesn't work with explicit implementations of `ITableEntity`?

Reply:
Yes, I'll add that in as a comment. As we both agree, this is really a sub-optimal solution - obviously it would be ideal if we could remove that limitation on the Storage SDK.